hooks/image/54-ek-launchers

```diff
@@ -0,0 +1,12 @@
+# Expose Endless Key launchers in the app grid (once a specific content pack has been downloaded)
+
+if [[ ! "${EIB_FLATPAK_REMOTE_FLATHUB_APPS}" =~ .*"org.endlessos.Key".* ]]; then
+  exit 0
+fi
+
+cat <<- EOF > /etc/systemd/user-environment-generators/62-endless-key.sh
```

Comment:
This hook is not running inside the image, but outside it. 
```suggestion
cat <<- EOF > "${OSTREE_DEPLOYMENT}/etc/systemd/user-environment-generators/62-endless-key.sh"
```

Reply:
I think you can avoid needing to backslash-escape the `$`s by single quoting the delimiter, i.e.

```suggestion
cat <<- 'EOF' > "${OSTREE_DEPLOYMENT}/etc/systemd/user-environment-generators/62-endless-key.sh"/etc/systemd/user-environment-generators/62-endless-key.sh"
```